programs/piaware-config/helpers.tcl

```diff
@@ -66,6 +66,9 @@ proc update_config_values {argv} {
 				puts stderr "Cleared setting for $key in [$result origin $key]"
 			} else {
 				puts stderr "Set $key to $displayVal in [$result origin $key]"
+				if {[::fa_piaware_config::is_network_setting $key]} {
```

Comment:
Let's check the `image-type` config setting here and only show the hint if this is a sdcard image

Reply:
(maybe this is an opportunity to implement something like `is_network_setting` but for all sdcard settings, and warn about configuring options that are sdcard-only on a non-sdcard install)

Reply:
Sure, let me go through the settings and determine which are SD card/non SD card

Reply:
I attempted to make use of the `-type` arg for this, (maybe supporting multiple types) but decided to go this simpler route. Let me look at this option again for SD card vs non-SD card settings